scripts/add-sumar-turnos.ts

```diff
@@ -0,0 +1,47 @@
+import { Agenda } from '../modules/turnos/schemas/agenda';
+import { sumar } from '../modules/obraSocial/schemas/sumar';
+import { Types } from 'mongoose';
+
+
+async function run(done) {
+    const ultimasAgendas = {
+        // Fechas entre las que hubo problema con pacientes SUMAR
+        $and: [
+            { horaInicio: { $gte: new Date('2021-07-15T00:00:00.000-03:00') } },
+            { horaInicio: { $lte: new Date('2021-08-04T00:00:00.000-03:00') } }
+        ]
+    };
+    const agendas = Agenda.find(ultimasAgendas).cursor({ batchSize: 100 });
+    const obraSocial = {
+        codigoPuco: null,
+        nombre: 'SUMAR',
+        financiador: 'SUMAR',
+    };
+    for await (const agenda of agendas) {
+        for await (const bloque of agenda.bloques) {
+            for await (const turno of bloque.turnos) {
```

Comment:
```suggestion
        for (const bloque of agenda.bloques) {
            for (const turno of bloque.turnos) {
```

Estan de más esos `await`, copy/paste detected!

Reply:
Me declaro culpable. Eliminados